templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -293,6 +293,8 @@ contents:
     type=internal
     EOF
           nmcli c load ${new_conn_file}
+          # Set interface metric in the static route case. For the DHCP route case, see below.
+          nmcli c mod ${ovs_interface} ipv4.route-metric "${bridge_metric}" ipv6.route-metric "${bridge_metric}"
```

Comment:
nit: `"${ovs_interface}"`
I know it's not an issue but is one less shellcheck warning and consistent with how we are using it in other places in this script.

Reply:
You're totally right. Changed, thnx